src/models/outbound/pispOTPValidate.model.ts

```diff
@@ -90,37 +91,35 @@ export class PISPOTPValidateModel
 
     this.logger.push({ channel }).info('onValidateOTP - subscribe to channel')
 
-    // eslint-disable-next-line no-async-promise-executor
-    return new Promise(async (resolve, reject) => {
-      let subId = 0
+    return deferredJob(this.pubSub, channel)
+    .init(async (channel) => {
+      const res = await this.thirdpartyRequests.patchConsentRequests(
+        consentRequestsRequestId,
+        { authToken: authToken }  as tpAPI.Schemas.ConsentRequestsIDPatchRequest,
+        toParticipantId
+      )
+
+      this.logger.push({ res, channel })
+        .log('ThirdpartyRequests.patchConsentRequests request call sent to peer, listening on response')
+      return Promise.resolve()
```

Comment:
we can drop this line, it is inside `async` function, so lack of `return` is transformed into Promise<void>.resolve()